package.json

```diff
@@ -3,107 +3,91 @@
   "version": "0.5.2",
```

Comment:
Bumping up the version number here is not only a matter of following best practices but also has some logic tied to it for flushing the cache from the users' browsers.

Reply:
would do these in another PR if it's ok?

Reply:
yeah, version bump should be a separate PR IMO

Reply:
release PR only for checking that all the commits about to be released are OK